packages/react-scripts/scripts/utils/scripts.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  start: 'BROWSER=none react-scripts start',
```

Comment:
Do not start browser on starting the project for development, as it is very annoying, and sometimes starts not the one used to development due to some corporate policies.